crates/hyperdrive-math/src/short/open.rs

```diff
@@ -157,14 +157,26 @@ impl State {
     /// will pay and comparing this to the amount of base the short will receive
     /// if the variable rate stays the same. The implied rate is just the ROI
     /// if the variable rate stays the same.
+    ///
+    /// To do this, we must adjust the variable rate $r_{adjusted}$ according to
+    /// the position duration $t$. Since we start off from a compounded APY and
+    /// output a compounded APY, the compounding frequency $f$ is simplified away,
+    /// so the adjusted rate will be:
+    ///
+    /// $$
```

Comment:
```
/// To do this, we must adjust the variable rate $r_{adjusted}$ according to
/// the annualized position duration $d$ (which equals `position_duration / 365 days`).
/// Since we start off from a compounded APY and output a compounded APY with an
/// normalizing constant of 1 year, the compounding frequency $f$ is simplified away,
/// so the adjusted rate will be:
///
/// $$
/// APR = f \cdot (( 1 + APY)^{\tfrac{1}{f}}  - 1)
/// $$
///
/// Therefore,
///
/// $$
/// \begin{aligned}
/// TPY &= (1 + \frac{APR}{f})^{d \cdot f}
/// &= (1 + APY)^{d} - 1
/// $$

Reply:
Beautiful. Updating this too!